sw/host/opentitanlib/src/image/manifest_ext.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use serde::{self, Deserialize, Serialize};
+use std::path::{Path, PathBuf};
+use zerocopy::AsBytes;
+
+use crate::crypto::spx::{self, SpxPublicKeyPart};
+use crate::image::manifest::*;
+use crate::image::manifest_def::le_bytes_to_word_arr;
+use crate::util::file::FromReader;
+use crate::util::num_de::HexEncoded;
+use crate::with_unknown;
+
+with_unknown! {
+    /// Known manifest extension variant IDs.
+    #[derive(Default)]
```

Comment:
What value is `default` for this type?  I assume `u32::ZERO` because the contained type is `u32`.

FWIW, you can also have `with_unknown` [derive default](https://github.com/lowRISC/opentitan/blob/master/sw/host/opentitanlib/src/util/unknown.rs#L261-L265) for you if you want it to be one of the enumerated values.  If you want it to be zero, then what you have works just fine.